libvast/src/system/source_command.cpp

```diff
@@ -99,11 +99,14 @@ caf::message source_command(const command& cmd, caf::actor_system& sys,
                ? caf::get<caf::actor>(node_opt)
                : caf::get<scope_linked_actor>(node_opt).get();
   VAST_DEBUG(&cmd, "got node");
-  /// Spawn an actor that takes care of CTRL+C and friends.
-  auto sig_mon = self->spawn<detached>(system::signal_monitor, 750ms,
-                                       actor{self});
+  // Start signal handler.
+  std::thread smon{[&] {
```

Comment:
Maybe give this thread a more descriptive name? The closest thing that comes to mind when reading out `smon` is "smores" 😉. Perhaps `thread` or `loop` would read a bit smoother.